src/server/scripts/Commands/cs_guild.cpp

```diff
@@ -33,6 +33,8 @@ EndScriptData */
 #include "ObjectMgr.h"
 #include "Player.h"
 #include "RBAC.h"
+#include "Define.h"         //  MONEY_COPPER_PER_GOLD
```

Comment:
MONEY_COPPER_PER_GOLD is not defined in Define.h (also Define.h is already included by almost every other header, making it redundant here), remove it